src/main/java/homework/task1/Task1.java

```diff
@@ -0,0 +1,26 @@
+package homework.task1;
+
+public class Task1 {
```

Comment:
Opt: please name apps by their names in homework, it will be easier for me to run

Reply:
done